sematic/db/migrations/20230217013045_add_user_id_to_runs.sql

```diff
@@ -0,0 +1,7 @@
+-- migrate:up
```

Comment:
Please add this file to the end of `schema.sql.sqlite`.

Reply:
Done.